tests/test_helper/many.bash

```diff
@@ -3,6 +3,22 @@ function pem() {
     echo "$PEM_ROOT/id-$1.pem"
 }
 
+# Print the X-coord of an Ed25519 public key
+function ed25519_x_coord() {
+    openssl pkey -in "$(pem "$1")" -text_pub -noout | grep "    " | awk '{printf("%s ",$0)} END { printf "\n" }' | sed '$s/ *//g' | tr -d ':'
+}
+
+# Return a CBOR encoded CoseKey created from a Ed25519 key.
+# Requires https://github.com/cabo/cbor-diag in your $PATH
+function key2cose() {
+  echo "{1: 1, 2: h'"$(identity_hex "$1")"', 3: -8, 4: [2], -1: 6, -2: h'"$(ed25519_x_coord "$1")"'}" | diag2cbor.rb | xxd -p -c 10000
```

Comment:
Where is `diag2cbor.rb` coming from?

Reply:
Look at the 2 lines above, as well as this PR description.